docs/pages/guides/customizing-metro.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Customizing Metro
+title: Metro bundler
+sidebar_title: Bundling with Metro
```

Comment:
we need to update "Customizing Webpack" to use the same naming convention if we change this: https://docs.expo.dev/guides/customizing-webpack/